src/features/room/search/ui/SearchPageRoomList.module.css

```
@@ -0,0 +1,21 @@
+.viewport {
+  --room-viewport-padding: 245px;
+  width: 100%;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)